receiver/mongodbatlasreceiver/metadata.yaml

```diff
@@ -0,0 +1,470 @@
+    description: Whether read into or written from
+    enum:
+      - read_into
+      - written_from
+  cache_status:
+    description: Cache status
+    enum:
+      - dirty
+      - used
+  ticket_type:
+    description: Type of ticket available
+    enum:
+      - available_reads
+      - available_writes
+  cursor_state:
+    description: Whether cursor is open or timed out
+    enum:
+      - timed_out
+      - open
+  memory_issue_type:
+    description: Type of memory issue encountered
+    enum:
+      - extra_info
+      - global_accesses_not_in_memory
+      - exceptions_thrown
+  global_lock_state:
+    description: Which queue is locked
+    enum:
+      - current_queue_total
+      - current_queue_readers
+      - current_queue_writers
+  btree_counter_type:
+    description: Database index effectiveness
+    enum:
+      - accesses
+      - hits
+      - misses
+  memory_state:
+    description: Memory usage type
+    enum:
+      - resident
+      - virtual
+      - mapped
+      - computed
+  direction:
+    description: Network traffic direction
+    enum:
+      - receive
+      - transmit
+  storage_status:
+    description: Views on database size
+    enum:
+      - total
+      - data_size
+      - index_size
+      - data_size_wo_system
+  operation:
+    description: Type of database operation
+    enum:
+      - cmd
+      - query
+      - update
+      - delete
+      - getmore
+      - insert
+  cluster_role:
+    description: Whether process is acting as replica or primary
+    enum:
+      - primary
+      - replica
+  document_status:
+    description: Status of documents in the database
+    enum:
+      - returned
+      - inserted
+      - updated
+      - deleted
+  execution_type:
+    description: Type of command
+    enum:
+      - reads
+      - writes
+      - commands
+  scanned_type:
+    description: Objects or indexes scanned during query
+    enum:
+      - index_items
+      - objects
+  disk_direction:
+    description: Measurement type for disk operation
+    enum:
+      - read
+      - write
+      - total
+  disk_status:
+    description: Disk measurement type
+    enum:
+      - free
+      - used
+  object_type:
+    description: MongoDB object type
+    enum:
+      - collection
+      - index
+      - extent
+      - object
+      - view
+      - storage
+      - data
+  oplog_type:
+    description: Oplog type
+    enum:
+      - slave_lag_master_time
+      - master_time
+      - master_lag_time_diff
+
+metrics:
+  mongodbatlas.process.asserts:
+    description: Number of assertions
+    extended_documentation: Aggregates MongoDB Metrics ASSERT_REGULAR, ASSERT_USER, ASSERT_MSG, ASSERT_WARNING
+    unit: "{assertions}/s"
+    labels: [assert_type]
+    data:
+      type: gauge
+  mongodbatlas.process.background_flush:
+    description: Amount of data flushed in the background
+    extended_documentation: MongoDB Metric BACKGROUND_FLUSH_AVG
+    unit: 1
+    data:
+      type: gauge
```

Comment:
Sum? Or this is per_second?

Reply:
Unfortunately I've found very little about this particular metric save the name. The documentation from MongoDB is what's here, I've seen reference in other documentation to it being in once place a counter, and in another the time the flush took in milliseconds. I've been unable to see this one live with ad-hoc testing to see what it reports, but I've seen enough questions of people using it for performance diagnoses that I don't want to ignore it.